Add training examples (#19)
iterorganization/IMAS-ParaView: docs/source/training_jintrac.rst

.. _`training_jintrac`:

JINTRAC case
------------
In this example, we will visualize the electron temperature GGD of the edge profiles IDS, as well as the 1D core profiles. We will use the psi grid from the equilibrium IDS to map the 1D electron temperature profiles onto a 2D grid. By doing this, we can visualize the electron temperature both in the edge plasma, as well as in the core in a single image.

> It is recommended to go through the steps below by hand, however if you do not want to follow the steps, a ParaView state file containing this example is made available :download:`here <pv_states/jintrac_state.pvsm>`.

Reviewer: This feels a bit paternal. Maybe rewrite to something like:

```suggestion
You can download the ParaView state file for this example :download:`here <pv_states/jintrac_state.pvsm>`. However, we recommend you to manually follow the steps outlined below.
```

Reviewer: Same comment applies to the other sections
